certified-connectors/Power Textor/apiDefinition.swagger.json

```diff
@@ -0,0 +1,6782 @@
+            "in": "body",
+            "required": true,
+            "schema": {
+              "type": "object",
+              "properties": {
+                "To": {
+                  "type": "array",
+                  "items": {
+                    "type": "string",
+                    "title": "To",
+                    "x-ms-dynamic-values": {
+                      "operationId": "GetContacts",
+                      "value-path": "Phone",
+                      "value-collection": "Data",
+                      "value-title": "Name"
+                    },
+                    "description": "Choose Multiple Contact Names from Dropdown"
+                  },
+                  "description": "To"
+                },
+                "Body": {
+                  "type": "string",
+                  "description": "Message body",
+                  "title": "Text Message"
+                },
+                "ReplySTOPToOptOut": {
+                  "type": "boolean",
+                  "description": "IF \u201cYes\u201d THEN populate \u201cSTOP to opt out\u201d in the end of text message, and the contact can unsubscribe to text messages by replying 'STOP' to the text message. If the user replies to the text message with 'START', he/she gets subscribed back to the service again and starts receiving the text messages again. \u2022 IF \u201cNo\u201d THEN do not populate the unsubscribe text in the end of the text message.",
+                  "title": "Reply STOP To Opt-Out",
+                  "enum": [true, false]
+                },
+                "GooglePlaceId": {
+                  "type": "string",
+                  "description": "Select Place name from dropdown to get feedback from customer",
+                  "title": "Place",
+                  "x-ms-dynamic-values": {
+                    "operationId": "GetPlaces",
+                    "value-path": "GooglePlaceId",
+                    "value-collection": "Data",
+                    "value-title": "Name"
+                  }
+                },
+                "ScheduledDate": {
+                  "type": "string",
+                  "description": "Select a date to send the text message on. Format should be MM/DD/YYYY",
+                  "title": "Schedule Date"
+                },
+                "ScheduledTime": {
+                  "type": "string",
+                  "description": "Select time to send the text message at. Time should be in Coordinated Universal Time. UTC. Format HH:mm AM/PM",
+                  "title": "Schedule Time (UTC)"
+                }
+              },
+              "required": [
+                "GooglePlaceId",
+                "ScheduledDate",
+                "ScheduledTime",
+                "To",
+                "Body"
+              ]
+            }
+          }
+        ]
+      }
+    },
+    "/[1:49 PM] Dua Ali https://api-staging-powertextor-v2.azurewebsites.net/api/messages/scheduledbulkreview": {},
```

Comment:
@MohsinAQ Empty operation definitions can be removed as they would not add any value to the connector. And moreover, it is a syntax error and would introduce issues in run time. Can you please share screenshots of successful flow runs of your connector.